pkgs/development/tools/pyp/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, fetchFromGitHub, python3Packages, pkgs }:
```

Comment:
No pkgs but explicit parameters such as jq and bc.

Reply:
Done, thanks. I felt a bit silly adding dependencies just because of two tests, but as long as they're not in the actual closure that's okay I suppose.